### ISSUE TITLE
Autocomplete: Lack of visual cue for results

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-design-system#4015]
This issue is from the accessibility audit of the Design System website by DAC in July 2024.

## DAC's description

* Page: [Home – GOV.UK Design System](https://design-system.service.gov.uk/)
* WCAG reference: none (Usability)
* Issue ID: DAC_Visual_Cues_Usability_03

When a user inputs data into the text field additional content will be revealed below. Our low vision analyst reported that when viewing the page within a highly magnified state it is difficult to determine if there is new content being added to the page and that it would be helpful if there were additional cues provided to inform them of this behaviour.

<img width="636" alt="Screenshot of the search box with it being unclear if any results have popped up or not" src="https://github.com/user-attachments/assets/bb1ff083-b273-4f70-857d-f5d4c658a528">

## DAC's proposed solution

We suggest implementing a chevron associated to the text input to indicate to users that there is additional content being displayed below.

## Other potential solutions

There might be other solutions.
If the chevron is always there, the component might be misunderstood to be a select box.
This needs some nuanced design thinking.

## Additional instances

This component appears on every page but is a single component, therefore only needs to be fixed once.
It uses the [accessible autocomplete component](https://github.com/alphagov/accessible-autocomplete).

## Needed roles

Designer, frontend developer

**Comment by hazalarpalikli** [comment]
Hello I have done some sketching on these. I need to catch-up with @CharlotteDowns and designers in the team to get a review on this. Designs are here if you want to have a look at in the meantime. https://www.figma.com/design/n8Jvs96qFFcKOw0qPe1se5/DAC-Accessibility-Issue?node-id=3-17&t=G7MQOd7VT5EKXhlg-1

**Comment by hazalarpalikli** [comment]
Hello, 

I have applied high contrast colours to the solution we are proposing. Attaching the images below, more could be found here as well [(https://www.figma.com/design/n8Jvs96qFFcKOw0qPe1se5/DAC-Accessibility-Issue?node-id=3-17&t=nYLbHvGnE2svut3I-1
)]

I have added some comments on the figma file. I have some concerns especially with the expanding card after the search. I am not sure if it's explicit enough and it's also allowing users to see the search below. 

<img width="592" alt="Screenshot 2024-09-05 at 16 41 41" src="https://github.com/user-attachments/assets/f1d0acb8-60e8-4782-95d0-585f785fff3c">
Alt: Design system landing page, search function with a focus state highlighted. 

<img width="586" alt="Screenshot 2024-09-05 at 16 42 03" src="https://github.com/user-attachments/assets/c61b6443-8d4a-43df-9af2-4c4fc274e2ef">
Alt: Design system landing page, search function with a focus state off

**Comment by Izabela-16** [comment]
@selfthinker if you can test it that would be amazing.
